secureli/actions/action.py

```diff
@@ -27,15 +27,15 @@ def __init__(
         echo: EchoAbstraction,
         language_analyzer: language_analyzer.LanguageAnalyzerService,
         language_support: language_support.LanguageSupportService,
-        scanner: ScannerService,
+        hooks_scanner: HooksScannerService,
```

Comment:
Just to leave context from a prior conversation during standup, I decided to rename this service so that it's more explicitly clear what it's used for, in contrast to any custom scans that are implemented (e.g., the PII scan). This scanner solely scans using external pre-commit hooks. Calling it a generic ScannerService didn't quite capture that nuance IMO.